### PR TITLE
Instructions for picking the correct infra. 

### PR DESCRIPTION
**Pull request by jgautam**
https://spectrocloud.atlassian.net/browse/DOC-89

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/584#" title="Author self-approved">jgautam</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/docs/content/04-clusters/1-new-clusters/05-edge.md
Comment (brathina-spectro) [inline packages/docs/content/04-clusters/1-new-clusters/05-edge.md:223]: osdsPerDevice should be indented under config

**Review by resmaa — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/584#" title="Author self-approved">jgautam</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/584#pullrequestreview-864435989" title="Approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/docs/content/04-clusters/1-new-clusters/05-edge.md
Comment (brathina-spectro) [inline packages/docs/content/04-clusters/1-new-clusters/05-edge.md:218]: This complete block should be under storage. Like 

```
storage: 
  useAllNodes: true
  useAllDevices: false
  deviceFilter: ^sd[d-f]
  config:
    osdsPerDevice: "1"
```

File: packages/docs/content/06-integrations/00-rook-ceph.md
Comment (brathina-spectro) [inline packages/docs/content/06-integrations/00-rook-ceph.md:27]: Copy the example from above comment

File: packages/docs/content/04-clusters/1-new-clusters/05-edge.md
Comment (brathina-spectro) [inline packages/docs/content/04-clusters/1-new-clusters/05-edge.md:219]: @resmaa This block needs to be indented as well just like the other one

File: packages/docs/content/06-integrations/00-rook-ceph.md
Comment (brathina-spectro) [inline packages/docs/content/06-integrations/00-rook-ceph.md:33]: Indentation is wrong, there's an extra space for osdsPerDevice

**Review by truegilb — APPROVED** [review]
Minor edits and approved

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/584#" title="Author self-approved">jgautam</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/584#pullrequestreview-864435989" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/584#pullrequestreview-867425084" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
